code/custom-zk-chain/scripts/depositBaseToken.ts

```diff
@@ -1,7 +1,12 @@
-import { ethers } from 'hardhat';
+import { ethers, network } from 'hardhat';
+import { Wallet, Provider } from 'zksync-ethers';
 
 async function main() {
-  const [wallet] = await ethers.getWallets();
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const config = network.config as any;
+  const l1Provider = new Provider(config.ethNetwork);
+  const l2Provider = new Provider(config.url);
+  const wallet = new Wallet(process.env.WALLET_PRIVATE_KEY!, l2Provider, l1Provider);
```

Comment:
why don't we want to use `ethers.getWallets()`, which returns the rich wallets from the node so we dont have to deal with PKs?